health-services/project-factory/src/server/validators/campaignValidators.ts

```
@@ -588,28 +650,42 @@
         throwError("COMMON", 400, "VALIDATION_ERROR", `Invalid Boundary Sheet. Root level Boundary not present in every row  of Sheet ${sheetName}`)
     }
 }
-function validateForDupicateRows(boundaryData: any[]) {
+function validateForDuplicateRows(boundaryData: any[]) {
+    // Step 1: Trim strings in all rows
     boundaryData = boundaryData.map(row =>
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)